### DISCUSSION TITLE
[A-L] (2023/24) Lezione 8 - Esercizio 4

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/algebra#178] [Esercizi - Viaggi/Piazza]
Dimostrare la seguente proprietà sull'anello $(\mathbb{Z},+,\cdot)$:
- $MCD(\pm\ 1,b)=1$

**Comment by Elia-Belli** [comment]
- E' ovvio che $1\geq 1$
- Sappiamo che tutti i numeri hanno come divisore ovvio $1$.
- Possiamo scrivere $1=1x_0+by_0$, con $x_0=1,y_0=0$
  se esiste $d': d'|1\ \land\ d'|b \Rightarrow d'|1x+by,\ \forall x,y \Rightarrow d'|1x_0+by_0 \Rightarrow d'|1 \Rightarrow d'=1$

$\Rightarrow MCD(\pm\ 1,b)=1$

**Comment by CiottoloMaggico** [comment]
![image](https://github.com/sapienzastudentsnetwork/algebra2324/assets/79638284/a86f8640-73bb-4634-8e53-b6c473a4c784)
